www/src/pages/migrating.mdx

```diff
@@ -112,8 +112,32 @@ via context to allow intermediate nesting of child components
 
 #### MenuItem
 
-- renamed from to `DropdownItem`
+- renamed to `Dropdown.Item`
```

Comment:
I think these should be listed like "renamed to `DropdownItem` (also exported on `Dropdown` as `Dropdown.Item`)" to be really clear, same for the Form components